document the start and stop cycle points (#191)
cylc/cylc-doc: src/glossary.rst

      * :term:`cycle point`
      * :term:`initial cycle point`

   start cycle point

>       The "start" :term:`cycle point` is the one at which the :term:`scheduler`
>       starts up at.

Reviewer: ```suggestion
      The "start" :term:`cycle point` is where the :term:`scheduler` starts.
```

      This may be before or after the :term:`initial cycle point`.

      See :ref:`start_stop_cycle_point` for more information.

      See also:

      * :ref:`start_stop_cycle_point`
      * :term:`cycle point`
      * :term:`stop cycle point`
      * :term:`initial cycle point`

   stop cycle point

>       The "stop" :term:`cycle point` is the one at which the :term:`scheduler`
>       shuts down.

Reviewer: ```suggestion
      The "stop" :term:`cycle point` is where the :term:`scheduler` shuts down.
```
